README.md

```diff
@@ -30,7 +30,7 @@
 Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to use the SDK
 
 ### Requirements
-*cordova-plugin-purchases* requires Xcode 13.2+ and minimum targets iOS 11.0+. The minimum Android version compatible is 4.4 (API level 19).
+*cordova-plugin-purchases* requires Xcode 15+ and minimum targets iOS 13.0+. The minimum Android version compatible is 5.0 (API level 21).
```

Comment:
Do these deployment target values actually get defined anywhere? I'm looking through the codebase and haven't been able to find where they would be defined 😅

Reply:
Not in the SDK, the apps need to specify it but I didn't see a way to specify min versions from the plugin unfortunately :(